lib/WorseReflection/Bridge/TolerantParser/Diagnostics/DocblockMissingExtendsTagProvider.php

```diff
@@ -138,7 +138,7 @@ class Foobar extends NeedGeneric
             assertion: function (Diagnostics $diagnostics): void {
                 Assert::assertCount(1, $diagnostics);
                 Assert::assertEquals(
-                    'Generic tag `@extends NeedGeneric` should be compatible with `@extends NeedGeneric<int>`',
+                    'Missing generic tag `@extends NeedGeneric<int>`',
```

Comment:
Another change between native and phpstan parser directly in test suite. As phpstan recognizes `@extends` tag without generic as invalid, the most simple solution is to annotate it with missing generic tag message